tests/UI/specs/OneClickUpdate_spec.js

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Piwik - free/libre analytics platform
+ *
+ * Screenshot tests for main, top and admin menus.
```

Comment:
The description is incorrect and maybe we should directly use a `Matomo` file header for new files...